src/screens/SendTokens.js

```diff
@@ -83,37 +119,122 @@ class SendTokens extends React.Component {
     return data;
   }
 
+  /**
+   * Add signature to each input and execute send transaction
+   */
+  onLedgerSuccess = (signatures) => {
+    const data = this.pendingData;
+    const keys = hathorLib.wallet.getWalletData().keys;
+    for (const [index, input] of data.inputs.entries()) {
+      const signature = Buffer.from(signatures[index]);
+      const keyIndex = keys[input.address].index;
+      const pubKey = hathorLib.wallet.getPublicKey(keyIndex);
+      input['data'] = hathorLib.transaction.createInputData(signature.slice(0, -2), pubKey);
```

Comment:
I think we should construct the transaction in the ledger library.

Reply:
I moved this `slice` operation to the Ledger code, so `SendTokens` doesn't need to know specifics about Ledger communication.  As for moving the rest of the code to the ledger lib, I'm not sure it would make a lot of sense. We'd only move the `for` loop.

Reply:
Ok.